.travis.yml

```diff
@@ -10,6 +10,9 @@ matrix:
     - php: 7.0
       env:
         - COMPOSER_FLAGS="--prefer-dist --no-interaction"
+    - php: 7.1
```

Comment:
Not sure if 5.6 should be also added to the matrix, possibly with --prefer-lowest as for 5.5?